presto-hive/src/test/java/com/facebook/presto/hive/TestHiveNativeWorkersQueries.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+import com.facebook.presto.testing.QueryRunner;
+import com.facebook.presto.tests.AbstractTestQueryFramework;
+import com.facebook.presto.tests.DistributedQueryRunner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Optional;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.airlift.tpch.TpchTable.NATION;
+
+public class TestHiveNativeWorkersQueries
+        extends AbstractTestQueryFramework
+{
+    protected TestHiveNativeWorkersQueries()
+    {
+        super(TestHiveNativeWorkersQueries::createQueryRunner);
+    }
+
+    private static QueryRunner createQueryRunner()
+            throws Exception
+    {
+        String prestoServerPath = System.getenv("PRESTO_SERVER");
```

Comment:
This is a small nit, but I think it would be preferable to make it clear these come from the environment, but having two methods, one which takes these as arguments, and an outer one which passes them in from the environment.